### PR TITLE
Added model fields to consultations and courses

### PR DESCRIPTION
**Pull request by daVitekPL**

### CODE REVIEW

**Review by qunabu — APPROVED** [review]

File: src/pages/Courses/index.tsx [hunk 1]
```diff
@@ -24,6 +24,10 @@ import { usePermissions } from '@/hooks/usePermissions';
 import { cloneCourse, course, exportCourse, removeCourse } from '@/services/escola-lms/course';
 import { createTableOrderObject, roundTo } from '@/utils/utils';
 import './style.less';
+import { history } from "@@/core/history";
+import useModelFields from "@/hooks/useModelFields";
+import { getLocale } from "@@/exports";
```
Comment (myslaf) [inline src/pages/Courses/index.tsx:29]: Change import to: "import { getLocale } from 'umi';"

File: src/pages/Consultations/fields.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import ModelFields from '@/components/ModelFields';
+
+const ConsultationFields: React.FC = () => {
+  return (
+    <PageContainer>
+      <ProCard
+        tabs={{
+          type: 'card',
+          activeKey: 'fields',
+          onChange: () => history.push('/other/consultations'),
```
Comment (myslaf) [inline src/pages/Consultations/fields.tsx:15]: Here in onChange function you should get key argument and set in url:
onChange: (key) => history.push('/other/consultations/${key}'), 
and add this in routing file /other/consultations/:tab or other name, then you can get this key from params by this used name in this example:
const { tab } = useParams<{ tab?: string }>();

File: src/pages/Consultations/index.tsx [hunk 3]
```diff
@@ -14,8 +14,13 @@ import type { ActionType, ProColumns } from '@ant-design/pro-table';
 import ProTable from '@ant-design/pro-table';
 import { Button, Popconfirm, Select, Tag, Tooltip, Typography, message } from 'antd';
 import { format } from 'date-fns';
-import React, { useCallback, useRef, useState } from 'react';
+import React, { useCallback, useMemo, useRef, useState } from 'react';
 import { FormattedMessage, Link, useIntl } from 'umi';
+import { history } from "@@/core/history";
+import ProCard from "@ant-design/pro-card";
+import useModelFields from "@/hooks/useModelFields";
+import { getLocale } from "@@/exports";
```
Comment (myslaf) [inline src/pages/Consultations/index.tsx:22]: Change import to: "import { getLocale } from 'umi';"

**Review by myslaf — APPROVED** [review]

**Review by victazzz — APPROVED** [review]

File: src/pages/Consultations/fields.tsx (hunk 2 above)
Comment (qunabu) [reply src/pages/Consultations/fields.tsx]: @myslaf @daVitekPL this is resolved ?

Comment (daVitekPL) [reply src/pages/Consultations/fields.tsx]: @qunabu I think so, this file is now removed and the logic moved to `src/pages/Consultations/index.tsx`

**Review by qunabu — APPROVED** [review]

**Pull request merged**